src/store/actions.ts

```diff
@@ -207,8 +207,15 @@ export default {
     const formFields = [...state.orderFormFields, { id: 'contents', type: 'text' }]
     const formData = {
       ...state.order,
-      ...{ contents: JSON.stringify(toCart(state)) },
-      ...{ updateDate: moment().toISOString() }
+      contents: JSON.stringify(toCart(state)),
+      updateDate: moment().toISOString()
+    }
+
+    if (state.context && state.context.context) {
```

Comment:
state.context.context looks like a bug ( waiting to happen to me ) maybe discuss with @fdlk if we can fix this ?

Reply:
Did so. Not in this PR :)

Reply:
The first context is the name of the vuex module, the second one is the name of the key in the state of the context in the vuex module. We can give them different names perhaps? But let's do it in separate PR